src/components/ParticipantList.jsx

```diff
@@ -53,6 +56,19 @@ const ParticipantList = () => {
 
   return (
     <Container maxWidth="sm">
+      <LoadingButton
+        data-cy="manage-participants-btn"
+        variant="contained"
+        margin="dense"
+        sx={{
+          backgroundColor: "#4C9074",
+          m: 5,
+        }}
+        onClick={() => setOpen(true)}
+      >
+        Manage Participant
```

Comment:
minor, but translation needed for `Manage Participant`

Reply:
Done